src/lib/battery/battery.cpp

```diff
@@ -55,7 +55,7 @@ Battery::Battery(int index, ModuleParams *parent, const int sample_interval_us,
 	const float expected_filter_dt = static_cast<float>(sample_interval_us) / 1_s;
 	_voltage_filter_v.setParameters(expected_filter_dt, 1.f);
 	_current_filter_a.setParameters(expected_filter_dt, .5f);
-	_current_average_filter_a.setParameters(expected_filter_dt, 25.f);
+	_current_average_filter_a.setParameters(expected_filter_dt, 50.f);
```

Comment:
Maybe we come to the conclusion to bump it even more, but let's for now keep it at 50